core/serialisation/src/test/java/uk/gov/gchq/gaffer/jsonSerialisation/JSONSerialiserTest.java

```diff
@@ -365,7 +366,16 @@ protected void deserialiseSecond(final Pair<Object, byte[]> pair) throws Seriali
 
     protected void serialiseFirst(final Pair<Object, byte[]> pair) throws SerialisationException {
         byte[] serialise = JSONSerialiser.serialise(pair.getFirst());
-        assertArrayEquals(pair.getSecond(), serialise);
+        List<Byte> expectedList = new ArrayList<>();
+        for (byte b : pair.getSecond()) {
+            expectedList.add(b);
+        }
+
+        List<Byte> actualList = new ArrayList<>();
```

Comment:
Does this need to happen? Can we instead use assertJ?

Reply:
So, the order of elements when ```serialiseFirst()``` method is returned is different and it causes the flakiness. Hence, I suggested the solution.

There aren't any assertions in assertJ which do not consider the order of the elements, so ArrayList was the option I considered. If you do have any suggestions I could try it out.

Reply:
Can you provide me with details on what order is not being preserved within the Java object?
Rather than making the test more "Loose" would it be better to make the serialisers more consistent?